modules/oe_content_publication/config/install/core.entity_view_display.node.oe_publication.default.yml

```diff
@@ -0,0 +1,49 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.oe_publication.oe_document
+    - field.field.node.oe_publication.oe_author
+    - field.field.node.oe_publication.oe_publication_date
+    - field.field.node.oe_publication.oe_publication_type
+    - field.field.node.oe_publication.oe_subject
+    - field.field.node.oe_publication.oe_summary
+    - field.field.node.oe_publication.oe_teaser
+    - node.type.oe_publication
+  module:
+    - datetime
+    - user
+id: node.oe_publication.default
+targetEntityType: node
+bundle: oe_publication
+mode: default
+content:
+  oe_document:
+    type: entity_reference_entity_view
+    weight: 2
+    label: above
+    settings:
+      view_mode: default
+      link: false
+    third_party_settings: {  }
+    region: content
+  oe_publication_date:
+    weight: 1
+    label: above
+    settings:
+      format_type: medium
+      timezone_override: ''
+    third_party_settings: {  }
+    type: datetime_default
+    region: content
+hidden:
+  links: true
+  oe_author: true
+  oe_content_content_owner: true
+  oe_content_legacy_link: true
+  oe_content_navigation_title: true
+  oe_content_short_title: true
+  oe_publication_type: true
+  oe_subject: true
+  oe_summary: true
+  oe_teaser: true
```

Comment:
Per here https://webgate.ec.europa.eu/CITnet/jira/browse/OPENEUROPA-1928
Fields Type and Summary should be displayed.